.github/workflows/test_python.yml

```diff
@@ -60,8 +65,17 @@ jobs:
     - run: buildozer init
     - name: SDK, NDK and p4a download
       run: |
-        sed "s/# android.accept_sdk_license = False/android.accept_sdk_license = True/" -i buildozer.spec
+        sed -i.bak "s/# android.accept_sdk_license = False/android.accept_sdk_license = True/" buildozer.spec
```

Comment:
sed in macOS doesn't work the same way as the one in Linux, the `-i.bak` is doing the trick to work on both